Tests/Admin/AdminTest.php

```diff
@@ -1523,19 +1524,11 @@ public function testGetSubject()
         $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
         $admin->setModelManager($modelManager);
 
-        $admin->setRequest(new Request(array('id' => 'azerty')));
```

Comment:
@greg0ire I still think the test should not be changed. Maybe the result, but not the test.